app/controllers/admin/api/services/proxy/policies_controller.rb

```diff
@@ -41,7 +40,7 @@ def update
       ApicastV2DeploymentService.new(@proxy).call(environment: :sandbox)
     end
 
-    policies_config = Proxy::PoliciesConfig.new(proxy.policies_config)
+    policies_config = proxy.policies_config
     policies_config.valid? # Needed to set errors if any
```

Comment:
I think this should be having the states from the `update_attribute`

It would be too late (attributes already saved in database) to display any error here...

Reply:
This was how it originally worked I believe. I will try to extend `invalid config - does not update policies` test to make sure invalid entries are not saved to database.

Reply:
I think I missed your point originally. It can be safely removed. Thanks!

I did some testing to see that it makes no difference. Also `test/integration/user-management-api/services/proxy/policies_test.rb` already tests that error messages are rendered.